aw/ViewControllers/Reaches/Detail/RunDetailTableViewController.swift

```diff
@@ -115,6 +115,11 @@ class RunDetailTableViewController: UITableViewController {
             return expandDescription ? UITableViewAutomaticDimension : 168
         case IndexPath(row: 0, section: 2):
             return 77
+        case IndexPath(row: 0, section: 3): // Gage info
+            if reach?.gageId != 0 {
+                return 44
```

Comment:
What do these numbers represent?

Reply:
They are the heights of those rows. If a run doesn’t have a gage set the height to 0 rather than 44.